### PR TITLE
Load example into long-running instance and periodically query

### PR DESCRIPTION
**Pull request by aelred**

### CODE REVIEW

**Comment by aelred** [comment]
!rtg - random failure in `BatchMutatorClientTest`

**Comment by aelred** [comment]
!rtg - timed out after 2 hours

File: scripts/periodic [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+PATH="/home/ubuntu/grakn:$PATH"
+
+while true; do
+        graql console -e 'match $x isa pokemon; get;';
```
Comment (fppt) [inline scripts/periodic]: This is not really periodic right? It's more a continuous and repeated query with no breaks?

Comment (aelred) [reply scripts/periodic]: Yes, I guess so! It's always querying. I should rename it.

**Pull request merged**